lib/extensions/postgres_cdc_rls/replication_poller.ex

```diff
@@ -45,12 +45,7 @@ defmodule Extensions.PostgresCdcRls.ReplicationPoller do
     tenant = args["id"]
 
     state = %{
-      backoff:
-        Backoff.new(
-          backoff_min: 100,
-          backoff_max: 5_000,
-          backoff_type: :rand_exp
-        ),
+      backoff: Backoff.new(backoff_min: 100, backoff_max: 5_000, backoff_type: :rand_exp),
```

Comment:
Since we started refactoring `ReplicationPooler`, let's replace `Backoff` with just `Enum.random/1`. Backoff is used here only to get a random timeout value

Reply:
will tackle this in another round, already a lot of changes happening